packages/neo-one-node-vm/src/__tests__/stackItem/StackItem.test.ts

```diff
@@ -273,9 +307,557 @@ describe('StackItem', () => {
     const arr = new ArrayStackItem([stackItem]);
     expect(arr.increment()).toEqual(2);
 
-    // [arr] NIP
+    // [arr] NIPF
     expect(stackItem.decrement()).toEqual(-1);
     // []
     expect(arr.decrement()).toEqual(-4);
   });
 });
+
+describe('Stack Item Tests', () => {
```

Comment:
Can you move these into separate files so that they can run in parallel? (I.e. 1 file per stackitem class type)